Userland/Libraries/LibWeb/Fetch/BodyInit.cpp

```diff
@@ -116,6 +116,11 @@ WebIDL::ExceptionOr<Infrastructure::BodyWithType> extract_body(JS::Realm& realm,
         }));
 
     // FIXME: 11. If source is a byte sequence, then set action to a step that returns source and length to source’s length.
+    // For now, do it synchronously.
+    VERIFY(source.has<ByteBuffer>());
```

Comment:
well…no need for `if` you `VERIFY` that just before, eh? 😅

Reply:
there's also at least one case in the visit call above where `source` ends up being a `Blob`.